sympy/benchmarks/mechanics/create_benchmark_tables.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
```

Comment:
Note for future. We don't typically commit any binary files into the main sympy repo, so the benchmarking.db should be committed. You would need to rebase to get rid of this file and force push back to this branch.